src/injected/textSelectorEngine.ts

```diff
@@ -79,5 +57,54 @@ function createMatcher(selector: string): Matcher {
     return text => re.test(text);
   }
   selector = selector.trim().toLowerCase();
-  return text => text.trim().toLowerCase() === selector;
+  return text => text.toLowerCase().includes(selector);
+}
+
+function queryInternal(root: SelectorRoot, matcher: Matcher): Element | undefined {
```

Comment:
I think this can be refactored into a generator to prevent duplicating the code for queryInternalAll.

Reply:
I don't think this duplication hurts.